docs/workflow/building/libraries/README.md

```diff
@@ -163,14 +163,14 @@ Under the src directory is a set of directories, each of which represents a part
 
 For example the src\libraries\System.Diagnostics.DiagnosticSource directory holds the source code for the System.Diagnostics.DiagnosticSource.dll assembly.
 
-You can build the DLL for System.Diagnostics.DiagnosticSource.dll by going to the `src\libraries\System.Diagnostics.DiagnosticsSource\src` directory and typing `dotnet msbuild`. The DLL ends up in `artifacts\bin\AnyOS.AnyCPU.Debug\System.Diagnostics.DiagnosticSource` as well as `artifacts\bin\runtime\[BuildConfiguration]`.
+You can build the DLL for System.Diagnostics.DiagnosticSource.dll by going to the `src\libraries\System.Diagnostics.DiagnosticsSource\src` directory and typing `dotnet msbuild`. The DLL ends up in `artifacts\bin\AnyOS.AnyCPU.Debug\System.Diagnostics.DiagnosticSource` as well as `artifacts\bin\runtime\[BuildSettings]`.
```

Comment:
Again, is `BuildSettings` a property defined?

Reply:
BuildSettings is the new name of buildConfiguration. We have reduced the use of buildConfiguration to just a concatenation of build properties. Coreclr uses buildConfiguration for debug and release. It will help  in having the same properties.
However here we can use the complete exapansion. (We can discuss about the name of the new property but it doesnt matter as much developers wont be using it.)